crates/nu-command/tests/commands/do_.rs

```diff
@@ -98,6 +98,15 @@ fn ignore_error_should_work_for_external_command() {
     assert_eq!(actual.out, "post");
 }
 
+#[test]
+fn ignore_error_with_complete_should_have_exit_code() {
+    let actual = nu!(cwd: ".", pipeline(
+        r#"(do -i { nu --testbin fail asdf } | complete).exit_code"#
+    ));
+
+    assert_eq!(actual.out, "0");
```

Comment:
I'm no expert, but I want to help.

Shouldn't this be non-zero?

Reply:
Ah, I get it now. The new implementation of `-i` ignores all errors, so it should be zero.

Reply:
On previous version, the value is 128, and I think the value might need to be 128

Reply:
Yes, thought they did stuff in https://github.com/nushell/nushell/pull/7122 and from what is said, especially in the documentation, it seems like `-i` is supposed to "ignore" either error, which would mean a zero, I guess. I don't know.

Reply:
Ok, so the exit code should be zero :-)